source/Octopus.Manager.Tentacle.Tests/ApplicationStartupSanityCheckFixture.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Autofac;
+using NUnit.Framework;
+using Octopus.Manager.Tentacle.Shell;
+
+namespace Octopus.Manager.Tentacle.Tests
+{
+    [TestFixture]
+    public class ApplicationStartupSanityCheckFixture
+    {
+        [Test]
+        public void ApplicationCanStartWithoutCrashing()
+        {
+            // If we can resolve the main view model without errors,
+            // then we can safely assume all the necessary components
+            // have been correctly registered in the IoC container
+            var container = App.ConfigureContainer();
+            _ = container.Resolve<ShellViewModel>();
```

Comment:
We should be resolving the `TentacleManagerModel` here, as that's the main model with all the dependencies.